framework/Samplers/AdaptiveDynamicEventTree.py

```diff
@@ -60,6 +60,20 @@ class cls.
 
     return inputSpecification
 
+  # @classmethod
+  # def getSolutionExportVariableNames(cls):
+  #   """
+  #     Compiles a list of acceptable SolutionExport variable options.
+  #     @ In, None
+  #     @ Out, ok, dict, {varName: manual description} for each solution export option
+  #   """
+  #   # cannot be determined before run-time due to variables and prefixes.
+  #   ok = super(AdaptiveDynamicEventTree, cls).getSolutionExportVariableNames()
+  #   new = {'{RESIDUUM}': 'RAVEN input name of module containing __residuumSign method; provides the evaluation of the function.'
+  #         }
+  #   ok.update(new)
+  #   return ok
+
```

Comment:
Do you need these lines?

Reply:
This is handled by LSS, so no longer needed  here. Removed.